.travis.yml

```diff
@@ -1,5 +1,11 @@
 language: objective-c
 
+sudo: false
+
+branches:
+  only:
+    - master
+
```

Comment:
`sudo: false` makes sure travis uses the [container-based infrastructure](https://docs.travis-ci.com/user/workers/container-based-infrastructure/). Which has:

* start up faster
* allow the use of caches for public repositories
* disallow the use of sudo, setuid and setgid executables

It's probably not a huge timesaver, but it's an option. The branches was something separate I was messing around with, I'll take it out.